payu/fsops.py

```diff
@@ -127,3 +127,7 @@ def patch_lustre_path(f_path):
             f_path = './' + f_path
 
     return f_path
+
+def dump_yaml(info, fname):
```

Comment:
I'm still not convinced this function should exist.

Reply:
It is now used more than once, you'd rather have the code inlined?

Reply:
I think so, yeah.  It's two lines and is basically the mechanism exposed by the module.

Not gonna die on this hill though, if you really like it then it's fine.